src/content/docs/apis/nerdgraph/examples/nerdgraph-entities-api-tutorial.mdx

```diff
@@ -646,7 +646,9 @@ You can manually delete any entity in your account by using the NerdGraph API. T
 mutation {
   entityDelete(guids: ["ENTITY_GUID_1", "ENTITY_GUID_2"]) {
     deletedEntities
-    failures
+    failures {
+      message
+    }
```

Comment:
This mutation could be improved by providing the GUIDS of failed entity deletions and possibly the error types.

```graphql
    failures {
      guid
      message
      type
    }
```